### PR TITLE
faster uuids conversion and commands split

### PR DESCRIPTION
**Pull request by NayXegp**
Faster conversion for uuids (7x faster)

Remove regex use for splitting commands (2x faster)

### CODE REVIEW

**Comment by md-5** [comment]
7x faster on what, 0.0001% of Bungee's CPU?

File: api/src/main/java/net/md_5/bungee/api/ServerPing.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
         private String name;
         private UUID uniqueId;
 
-        private static final UUID md5UUID = Util.getUUID( "af74a02d19cb445bb07f6866a861f783" );
+        private static final UUID md5UUID = UUID.fromString( "af74a02d-19cb-445b-b07f-6866a861f783" );
```
Comment (md-5) [inline api/src/main/java/net/md_5/bungee/api/ServerPing.java]: Why change this?

**Comment by md-5** [comment]
Also yeah please do post benchmark for both

**Comment by NayXegp** [comment]
Tested with JMH, the score of the actual code is 1416366 and the score of the new code is 7166952

**Comment by md-5** [comment]
Can you include the JMH tests though

File: api/src/main/java/net/md_5/bungee/api/ServerPing.java (hunk 1 above)
Comment (md-5) [reply api/src/main/java/net/md_5/bungee/api/ServerPing.java]: ^

**Comment by NayXegp** [comment]
This is the JMH tests I used: https://gist.github.com/NayXegp/0b2d350b4e1a89c1fb788e26f662222b

**Comment by md-5** [comment]
Thanks

**Pull request closed without merging**